docs/user/openstack/troubleshooting.md

```diff
@@ -15,6 +15,18 @@ OpenStack CLI tools should be installed, then:
 
 Destroying the cluster has been noticed to [sometimes fail](https://github.com/openshift/installer/issues/1985). We are working on patching this, but in a mean time the workaround is to simply restart the destroying process of the cluster.
 
+## Machine has ERROR state
+
+This could be because the machine's instance was accidentally destroyed and the cluster API provider cannot recreate it.
+
+You can check the status of machines with the help of the command `oc get machines -n openshift-machine-api`.
+
+If a machine has the error state, you should delete it manually with `oc delete machine -n openshift-machine-api <machine_name>` command. The operation can take up to 5 minutes, during which time the machine will be gracefully deleted and all resources returned to the pool.
```

Comment:
Hmm, what if the users deletes the control-plane machine... if you delete that object, no new machine object will be created as it is not parented by a machineset.

Reply:
added this section too